source/result.js

```diff
@@ -101,12 +98,14 @@ Object.assign(Ok.prototype, {
     return this.value;
   },
 
-  map(λ) {
-    return transformResult(() => λ(this.value), Ok);
-  },
-
-  flatMap(λ) {
```

Comment:
Same as above

Reply:
that's why I chose to have aliases

Reply:
`flatMap` is not supposed to do any trick, and the management of Promises is already more than it should do